ci-operator/step-registry/acm/interop-p2p/spoke-upgrade-healthcheck/acm-interop-p2p-spoke-upgrade-healthcheck-commands.sh

```
@@ -0,0 +1,471 @@
+            echo "${updating_mcp}"
+            rm -f "${tmp_output}"
+            return 1
+        fi
+    else
+        : "Did not run 'oc get machineconfigpools' successfully"
+        rm -f "${tmp_output}"
+        return 1
+    fi
+
+    # Do not check UPDATED on purpose, because some paused mcp would not update itself until unpaused
+    oc get machineconfigpools -o custom-columns=NAME:metadata.name,CONFIG:spec.configuration.name,UPDATING:status.conditions[?\(@.type==\"Updating\"\)].status,DEGRADED:status.conditions[?\(@.type==\"Degraded\"\)].status,DEGRADEDMACHINECOUNT:status.degradedMachineCount --no-headers > "${tmp_output}" || true
```

Shellcheck (0.11.0)

[info] Ranges can only match single chars (mentioned due to duplicates).

(SC2102)